components/header.js

```diff
@@ -25,13 +30,13 @@ export default class Header extends Component {
                   </div> {/* /navbar-header */}
                   <div id="nav-bar" className={`${this.state.collapsed ? 'collapse' : ''} navbar-collapse`}>
                     <ul className="nav navbar-nav navbar-right navbar-links">
-                      <li><a data-scroll="true" href="#about">About</a></li>
-                      <li><a data-scroll="true" href="#services">Services</a></li>
-                      <li><a data-scroll="true" href="#our-work">Our Work</a></li>
-                      <li><a data-scroll="true" href="#techstack">Tech</a></li>
-                      <li><a data-scroll="true" href="#team">Team</a></li>
-                      <li><a href="https://blog.z-dev.io/">Blog</a></li>
-                      <li><a data-scroll="true" href="#contact">Contact</a></li>
+                      <li><a data-scroll="true" href="#about" onClick={this.onClick.bind(this)}>About</a></li>
```

Comment:
two things. 

1) might be possible to just set onClick on ul which will fire when anything inside is clicked?
2) you never need to use bind.

You can always do something like:
`onClick={() => this.onClick()}`

Which is much more intuitive. Although why this is a problem in the first place, is not very intuitive tbh